src/app/tests/suites/certification/Test_TC_OO_2_3.yaml

```diff
@@ -817,8 +821,8 @@ tests:
       PICS: OO.S.A4001 && OO.S.C42.Rsp
       response:
           constraints:
-              minValue: 170
-              maxValue: 230
+              minValue: 200 - PIXIT.OO.MaxCommunicationTurnaround
+              maxValue: 200 + PIXIT.OO.MaxCommunicationTurnaround
```

Comment:
I think that 15% tolerance applies here, it elapsed 10 seconds since the countdown starts, 15% of 100 is 15, so in total `215`.
So in total I think it should be `215`.

Reply:
Updated